docs/_data/fuzzers.yaml

```diff
@@ -0,0 +1,169 @@
+  desc: AFL modifications.
+  year: 2020
+  url:
+  dblp:
+  repo: https://github.com/alifahmed/aflmod
+  dir: fastcgs_lm
+  issue:
+  experimental: true
+
+- name: Ankou
+  desc: Guiding Grey-box Fuzzing towards Combinatorial Difference.
+  year: 2020
+  url: https://www.jiliac.com/files/ankou-icse2020.pdf
+  dblp:
+  repo: https://github.com/SoftSec-KAIST/Ankou
+  dir:
+  issue: https://github.com/google/fuzzbench/issues/198
+
+- name: AFL++
+  desc: afl++ is afl with community patches.
+  year: 2019
+  url: https://aflplus.plus
+  dblp:
+  repo: https://github.com/AFLplusplus/AFLplusplus
+  dir: aflplusplus
+  issue:
+
+- name: MOpt-AFL
+  desc: AFL with Particle Swarm Optimization (PSO).
+  year: 2019
+  url: https://www.usenix.org/conference/usenixsecurity19/presentation/lyu
+  dblp: conf/uss/LyuJZLLSB19
+  repo: https://github.com/puppet-meteor/MOpt-AFL
+  dir: mopt
+  issue:
+
+- name: AFLSmart
+  desc: AFLSmart is a smart input-structure aware greybox fuzzer.
+  year: 2019
+  url: https://thuanpv.github.io/publications/TSE19_aflsmart.pdf
+  dblp: journals/corr/abs-1811-09447
+  repo: https://github.com/aflsmart/aflsmart
+  dir: aflsmart
+  issue:
+
+- name: Eclipser
+  desc: Grey-box Concolic Testing on Binary Code.
+  year: 2019
+  url: https://softsec.kaist.ac.kr/~jschoi/data/icse2019.pdf
+  dblp: conf/icse/0002JHC19
+  repo: https://github.com/SoftSec-KAIST/Eclipser
+  dir: eclipser
+  issue:
+
+- name: FairFuzz
+  desc: A targeted mutation strategy for increasing greybox fuzz testing coverage.
+  year: 2018
+  url: https://www.carolemieux.com/fairfuzz-ase18.pdf
+  dblp: conf/kbse/LemieuxS18
+  repo: https://github.com/carolemieux/afl-rb
+  dir: fairfuzz
+  issue:
+
+- name: Angora
+  desc: Angora is a mutation-based fuzzer.
+  year: 2018
+  url: https://web.cs.ucdavis.edu/~hchen/paper/chen2018angora.pdf
+  dblp: conf/sp/ChenC18
+  repo: https://github.com/AngoraFuzzer/Angora
+  dir:
+  issue:
+
+- name: Vuzzer
+  desc: Application-aware Evolutionary Fuzzing.
+  year: 2017
+  url: https://www.cs.vu.nl/~giuffrida/papers/vuzzer-ndss-2017.pdf
+  dblp: conf/ndss/0001JKCGB17
+  repo: https://github.com/vusec/vuzzer
+  dir:
+  issue:
+
+- name: AFLFast
+  desc: Coverage-based Greybox Fuzzing as Markov Chain.
+  year: 2016
+  url: https://mboehme.github.io/paper/TSE18.pdf
+  dblp: conf/ccs/BohmePR16
+  repo: https://github.com/mboehme/aflfast
+  dir: aflfast
+  issue:
+
+- name: laf-intel
+  desc: LLVM instrumentation passes that help fuzzers.
+  year: 2016
+  url: https://lafintel.wordpress.com
+  dblp:
+  repo: https://gitlab.com/laf-intel/laf-llvm-pass
+  dir: lafintel
+  issue:
+
+- name: Driller
+  desc: Combines AFL and angr.
+  year: 2016
+  url: https://sites.cs.ucsb.edu/~vigna/publications/2016_NDSS_Driller.pdf
+  dblp: conf/ndss/StephensGSDWCSK16
+  repo: https://github.com/shellphish/driller
+  dir:
+  issue:
+
+- name: HonggFuzz
+  desc: A security oriented, feedback-driven, evolutionary, easy-to-use fuzzer.
+  year: 2015
```

Comment:
Not a big deal but I thought Honggfuzz predates AFL but  adopted coverage guided fuzzing later.